tests/builders/test_fully_connected.py

```diff
@@ -69,6 +99,7 @@ def fully_connected_module_match_cfg(
     input_features=st.integers(min_value=1, max_value=32),
     output_features=st.integers(min_value=1, max_value=32),
 )
+@settings(deadline=3000)
```

Comment:
3 seconds is long for each *individual* invocation of the test (which could be ran many times). Is this necessary? How far can it be reduced so the test suite time is kept short?

Reply:
I've reduced the max number of layers from `8 -> 3` so timings are much more reasonable. This settings is now removed.